java-client/flight/src/main/java/io/deephaven/client/impl/VectorHelper.java

```diff
@@ -30,6 +23,14 @@ public static void fill(TinyIntVector vector, byte[] array, int offset, int len)
         vector.setValueCount(len);
     }
 
+    public static void fill(BitVector vector, byte[] array, int offset, int len) {
+        vector.allocateNew(len);
+        for (int i = 0; i < len; i++) {
+            vector.set(i, array[offset + i] != Util.NULL_BOOL ? 1 : 0, array[offset + i] == Util.TRUE_BOOL ? 1 : 0);
+        }
+        vector.setValueCount(len);
+    }
```

Comment:
Instead of exposing the constants, I think it might be better to present a new interface for iterating/accessing items through `BooleanArray`.

Maybe it's as simple as:

```java
public final class BooleanArray extends PrimitiveArrayBase<Boolean> {
    ...
    public Boolean value(int index) {
        ...
    }
```
?

It's also not unreasonable to think this should be generic on `io.deephaven.qst.array.Array<T>`.

Reply:
We might additionally consider adding:

```java
public boolean isNull(int index) {
...
}
```

for users who want to iterate the primitive arrays without needing to go through a potentially boxing / unboxing.